### PR TITLE
perf-report-creator: update regex and plot.go to include all graphs in one index.html

### LINKED ISSUE #2741: sig-scale: bug in regex used for scrapping data
In the April 27th community call, it was discussed that the regex used for scraping the data in `perf-report-creator` is incorrect.

It assumes that the data for each resource is printed after the statements for the jobs, but instead it is printed before.  
```
[sig-performance][Serial] Control Plane Performance Density Testing Density test [small] create a batch of 100 VMIs should sucessfully create all VMIS
```

The regex needs to be fixed to get accurate results. cc @rthallisey

### PR DESCRIPTION
**Pull request by alaypatel07**
Fixes #2741 

cc @rthallisey

### CODE REVIEW

**Comment by alaypatel07** [comment]
The data crunched from this bug-fix is here https://alaypatel07.github.io/sig-scale-health.github.io/week-2/vmi/

**Comment by rthallisey** [comment]
seems ok.
/lgtm

**Review by brianmcarey — APPROVED** [review]
thanks @alaypatel07 - looks good!

/approve

Just one nit comment inline

File: robots/cmd/perf-report-creator/main.go [hunk 1]
```diff
@@ -377,12 +383,12 @@ func readLinesAndMatchRegex(file io.Reader, jsonStartRegex string) (string, erro
 				break
 			}
 			// This is needed to make sure the starting of match is json object `{`
-			lines = lines[3:]
-			jsonText := strings.ReplaceAll(strings.Join(lines, ""), "\\n", " ")
+			//lines = lines[3:]
```
[the remark this answers is not in the document]
Comment (alaypatel07) [reply robots/cmd/perf-report-creator/main.go:386]: yes, I'll be sure to clean this up in followup if thats okay

**Pull request merged**